locale/en/docs/guides/simple-profiling.md

```diff
@@ -72,6 +77,8 @@ app.get('/auth', (req, res) => {
     res.sendStatus(401);
   }
 });
+
+app.listen(port, () => console.log(`profiling example app running on port ${port}`))
```

Comment:
Shortened the message to keep it < 80 lines, but feel free to keep the older message and format it differently.

```suggestion
app.listen(port, () => console.log(`Running on port ${port}`));
```